### DISCUSSION TITLE
Unable to make project public

### DISCUSSION THREAD

**Discussion by rhys-vdw** [community/community#23560] [New to GitHub]
<p>Following guide here: <a href="https://docs.github.com/en/issues/organizing-your-work-with-project-boards/managing-project-boards/changing-project-board-visibility" class="inline-onebox" rel="noopener nofollow ugc">Changing project board visibility - GitHub Docs</a></p>
<p>There is no “settings” option on <a href="https://github.com/rhys-vdw/enter-the-chronosphere/projects/1" rel="noopener nofollow ugc">my project</a> (currently private).</p>
<p>Instead I have the “edit” option, which does not have the option to make the board public.</p>
<p><img src="https://user-images.githubusercontent.com/822105/181077900-64ac2b83-8f40-44b0-908b-bd4d9042bf79.png" alt="image" data-base62-sha1="bbOCVbKUBPObg9m8bC664M5Vi5y" width="240" height="165"></p>
<p>Has this option been removed or is the documentation incomplete/out of date?</p>

**Comment by Zerotask** [comment]
<p><a href="https://docs.github.com/en/github/administering-a-repository/managing-repository-settings/setting-repository-visibility" class="onebox" target="_blank" rel="noopener nofollow ugc">https://docs.github.com/en/github/administering-a-repository/managing-repository-settings/setting-repository-visibility</a></p>

**Comment by rhys-vdw** [comment]
<p>I’m interested in making the <em>project</em> public, not the repository.</p>

**Comment by rhys-vdw** [comment]
<p>Still looking for clarification on this one.</p>

**Comment by rhys-vdw** [comment, marked as answer]
<p>Seems this is addressed by the new projects changes!</p>
